### PR TITLE
[T3A5][W13-A2]YongLinHan

### PR DESCRIPTION
**Pull request by ylhlh**
Ready for review

### CODE REVIEW

**Comment by okkhoy** [comment]
@ylhlh in general this is a good effort.

one minor thing to be aware of: be consistent with the way you write the header comments
adapt the style as used in the code given to you.

please ack and close this PR once you have read the comments.

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    
+    private static String addDecorationsToText(String... message) {
+        final StringBuilder toShow = new StringBuilder();
+        for (String m : message) {
+            toShow.append(LINE_PREFIX + m.replace("\n", LS + LINE_PREFIX)).append("\n");
+        }
+        return toShow.toString();
+    }
+    
+    /** Formats a list of strings as a viewable indexed list. */
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java]: please be consistent with the header comment style. better to follow multi-line comment style throughout

Comment (ylhlh) [inline src/seedu/addressbook/ui/Formatter.java]: Actually I just copy pasted all the comments from the text UI file haha, didn't change any of them. But I agree that the comment style is different and should be fixed.

**Comment by ylhlh** [comment]
Is this code accepted? Or do you want me to resubmit another pull request with the changes?

**Pull request closed without merging**